src/libs/KeyboardShortcut/index.ts

```diff
@@ -130,12 +132,12 @@ function subscribe(
     key: string,
     callback: () => void,
     descriptionKey: string,
-    modifiers: string[] = ['shift'],
+    modifiers: ShortcutModifiers = ['CTRL'],
```

Comment:
@kubabutkiewicz Why did you change the default value?

Reply:
@VickyStash Because modifiers are declared [here](https://github.com/Expensify/App/blob/a31933a049b9a9aa4222a4d3edab19b54a4555b7/src/CONST.ts#L302C1-L302C1) and there is no shortcut with only shift key modifier so it doesn't make sense to default it to that key